### PR TITLE
DOC: release notes for v0.6.2

### PR DESCRIPTION
**Pull request by mrakitin**
Release notes for the new patch release.

### CODE REVIEW

**Comment by mrakitin** [comment]
I've also updated the PyPI publishing workflow to employ the OpenID Connect approach (no secrets are required!)

image

**Comment by mrakitin** [comment]
@AbbyGi, @BriannaRomasky, this is ready for your review! I've restarted a single failing test, the failure was random, not related to the code changes. Please review/merge today, so that I could release the new version.

File: docs/source/release-history.rst [hunk 1]
```diff
@@ -2,6 +2,36 @@
...
+  automatically install it, and imports of ``xraylib`` will print the
+  information about that missing library, even for irrelevant code (like
+  MAD-X). Adding the package as a dependency explicitly silences the messages.
+
+- Remove pinning of the ``urllib3`` package as the issue is resolved on the
+  ``vcrpy`` side.
+
+Documentation
+.............
+- Update databroker configuration instructions for Windows.
```
Comment (mrakitin) [inline docs/source/release-history.rst:27]:
```suggestion
- Updated databroker configuration instructions for Windows.
```

File: docs/source/release-history.rst [hunk 2]
```diff
@@ -2,6 +2,36 @@
 Release History
 ===============
 
+v0.6.2 (2023-06-09)
+-------------------
+This is a maintenance release with small packaging and documentation updates.
```
Comment (mrakitin) [inline docs/source/release-history.rst:7]:
```suggestion
This is a maintenance release with small API, tests, packaging, and documentation updates.
```

**Comment by mrakitin** [comment]
Let's make sure the CI runs succeed before merging. Thanks!

**Review by BriannaRomasky — APPROVED** [review]
Looks good!

**Comment by mrakitin** [comment]
Thank you, @AbbyGi, @BriannaRomasky!

**Pull request merged**